### ISSUE TITLE
v4.9.0 breaks eth_signTypedData

### ISSUE THREAD

**Issue by shaharsol** [MetaMask/metamask-extension#5014]
my code that used to work:
```
web3.currentProvider.sendAsync({
      method: 'eth_signTypedData',
      params: [params, App.account],
      from: App.account,
    }, function (err, result) {
      callback(err,result)
    })
```
now I get this message in the console but nothing happens:

![screen shot 2018-08-09 at 6 52 37 pm](https://user-images.githubusercontent.com/672778/43910470-94787650-9c05-11e8-91e0-bdbe8f3d8788.png)

**Comment by shaharsol** [comment]
It's sporadic, but in some cases after I click the button that fires this function and then explicitly click the metamask fox on the toolbar, I get this window, which I'd expected to pop up:

![screen shot 2018-08-09 at 7 02 45 pm](https://user-images.githubusercontent.com/672778/43911034-0e6f0fae-9c07-11e8-8928-19a9bf3eaaf2.png)


yet, even after I sign, the callback is not called within my js code.

**Comment by shaharsol** [comment]
on another desktop, when i click "sign" i see an endless loading gif, and I managed to grab this from the "old ui" dev tools:

![metamask1](https://user-images.githubusercontent.com/672778/43912006-a8a28162-9c09-11e8-916a-eba18cd9a328.png)

I bet the undefined EIP712Domain has something to do with the error?

**Comment by bitpshr** [comment]
Hi @shaharsol. This was caused by a premature version bump of a few of our dependencies and will be fixed by #4989. We plan to do a patch release shortly thereafter. Your existing dapp code should begin to work again at that time. Thanks for the issue report and for your patience!
